package.json

```diff
@@ -23,7 +23,7 @@
     "lodash-node": "^2.4.1",
     "object-fit-videos": "^1.0.3",
     "ophan-tracker-js": "1.3.9",
-    "prebid.js": "https://github.com/guardian/Prebid.js.git#a549ab0",
+    "prebid.js": "https://github.com/guardian/Prebid.js.git#5b3652a",
```

Comment:
this build of Prebid contains the consentManagement module 😁 

So the bids can now have the consent data added:

<img width="1219" alt="picture 56" src="https://user-images.githubusercontent.com/8607683/40517045-54435488-5fab-11e8-8fc7-6ddce954b087.png">